handbook/sales/README.md

```diff
@@ -123,9 +123,11 @@ Sometimes there is a change in the champion within the customer's organization.
     3. Offer training to get them up to speed.
     4. Provide a white glove experience.
 
-### Schedule a customer call
-To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc) with a Fleet customer, use the ["Customer meeting" Calendly link](https://calendly.com/fleetdm/customer).
-- **Documenting a customer call:** When we do prospect calls, add the customer's name in both the google doc title and the heading, ex. "Charlie (Fleet)."  This makes it easier when searching for the document later. 
+### Schedule a customer or prospect call and add a Solutions Consultant (SC)
```

Comment:
Mike: Should this responsibility be named “Schedule a prospect call”? "Schedule a customer call" would live on the customer success page.

Reply:
@Sampfluger88 SCs take both pre and post sales calls with prospects and customers.

Reply:
Agreed, The “Schedule a prospect call” would live on the customer success page.